packages/angular_devkit/build_angular/src/builders/browser-esbuild/schema.json

```diff
@@ -0,0 +1,542 @@
+{
```

Comment:
Note: while technically we could use `allOf` ex:
```js
{
  "allOf": [
    {
      "properties": {
        "externalDependencies": {...}
    },
    { "$ref": "../browser/schema.json" }
  ]
}
```

quicktype doesn't merge this properly. See: https://github.com/quicktype/quicktype/issues/1840